Gulpfile.js

```diff
@@ -165,7 +185,7 @@ gulp.task('lint', function () {
             'Gulpfile.js'
         ])
         .pipe(eslint())
-        .pipe(eslint.format())
+        .pipe(eslint.format(process.env.ESLINT_FORMATTER))
```

Comment:
Why does it need? We dont't use a custom formatter passed via environment variable.

Reply:
I've found that WebStorm can parse links outputted by [unix](https://eslint.org/docs/user-guide/formatters/#unix) or [visualstudio](https://eslint.org/docs/user-guide/formatters/#visualstudio) formatters and navigate to the specific line and column where the error was emitted. 

But the default [stylish](https://eslint.org/docs/user-guide/formatters/#stylish) is more readable for plain text, so I would keep it as the default one, and use the `unix` formatter only in WebStorm.

Reply:
Ok